deployment/scripts/install.sh

```diff
@@ -286,22 +286,22 @@ fi
 echo "(Re-)Creating deployed objects"
 if [ "${KEEP_SUPPORT}" != true ]; then
 	oc process logging-support-template | oc delete -f - || :
-	oc delete serviceaccount,service,route --selector logging-infra=support
+	oc delete imagestream,service,route --selector logging-infra=support
```

Comment:
:) yeah exactly, let users know that image streams will now be recreated and if you don't want this to happen (for some reason) you can remove the label to prevent that.